### ISSUE TITLE
after deleting small talk pre built category, will get crash/bug after importing it again

### ISSUE THREAD

**Issue by RussellSPG** [samtecspg/articulate#1053]
### Issues
Deleted small talk pre made categories after delting all actions, keywords etc.

Then added small talk pre-made category again, and got blank page with this console error:
```
The above error occurred in task Fe
    created by takeLatest(app/CategoryPage/IMPORT_CATEGORY, Fe)
    created by Le
Tasks cancelled due to error:
takeLatest(app/CategoryPage/IMPORT_CATEGORY, Fe)
takeLatest(app/CategoryPage/LOAD_PREBUILT_CATEGORIES, ke
```
This is on master

**Comment by wrathagom** [comment]
@dakavisa when I did this I:

1. Imported the Small talk category
2. Deleted all of the sayings
3. Deleted all of the keywords
4. Tried to import small talk again

I also got a blank page. The error was due tot he fact that I never deleted the actions.

```
{
    "statusCode": 404,
    "error": "Not Found",
    "message": "The Agent already has Action with the name(s) = Greet, Goodbye, Gratitude, Who Am I, How Old, Capabilities, Busy, Are you there, Ask Mood, Answer Mood, You Are Funny, I like you, By my Friend, Agree, You Suck, You Awesome, You're Welcome, Marriage Proposal, Sad."
}
```

So it's valid not to re-import until those are deleted, but we should handle the error rather than just erroring to a blank page.

**Comment by dakavisa** [comment]
Closed by 072bcf59d9b8e012d02a9675e97774508b868d14
